api/proto/runme/parser/v1/parser.proto

```diff
@@ -54,18 +55,53 @@ enum CellKind {
   CELL_KIND_UNSPECIFIED = 0;
   CELL_KIND_MARKUP = 1;
   CELL_KIND_CODE = 2;
+  CELL_KIND_DOC_RESULTS = 3; // todo(sebastian): is this needed?
+}
+
+enum CellRole {
+  CELL_ROLE_UNSPECIFIED = 0;
+  CELL_ROLE_USER = 1;
+  CELL_ROLE_ASSISTANT = 2;
 }
 
 message Cell {
+  // CellKind is the type of cell, e.g. markup or code.
   CellKind kind = 1;
+
+  // Value is the contents of the cell.
   string value = 2;
+
+  // LanguageID is the language of the cell.
   string language_id = 3;
+
+  // Metadata is additional metadata about the cell in k/v format.
   map<string, string> metadata = 4;
+
+  // TextRange is the range of the cell covering in the markdown source.
   TextRange text_range = 5;
+
+  // Outputs are the outputs of the cell, e.g. stdout, stderr, or other media types.
   repeated CellOutput outputs = 6;
+
+  // ExecutionSummary is the summary of the cell execution, e.g. success/failure and execution time.
   CellExecutionSummary execution_summary = 7;
+
+  /* AGENT SPECIFIC FIELDS (porting from Block) */
```

Comment:
@jlewi, I've managed to get the first full test suite passing with Runme-ported protos.

Please check it out and let me know your thoughts on the modeling. The most notable difference is that I renamed `file_search_results` into `doc_results`. If we want to retain that it's the result of a file search, we could add a property to every result.

We'll probably keep the 100 offset and "settle" the sequence in a future `v2` of the parser.